cc_plugin_ncei/ncei_base.py

```diff
@@ -587,8 +603,8 @@ def check_geophysical(self, dataset):
             test_ctx.assert_true(getattr(ncvar, 'long_name', '') != '', 'long_name should exist and not be empty')
             if not hasattr(ncvar, 'standard_name'):
                 test_ctx.assert_true(getattr(ncvar, 'nodc_name', '') != '', 'nodc_name should exist and not be empty')
-            test_ctx.assert_true(getattr(ncvar, 'valid_min', '') != '', 'valid_min should exist and not be empty')
-            test_ctx.assert_true(getattr(ncvar, 'valid_max', '') != '', 'valid_max should exist and not be empty')
+                test_ctx.assert_true(getattr(ncvar, 'valid_min', '') != '', 'valid_min should exist and not be empty or valid_range should be defined')
+                test_ctx.assert_true(getattr(ncvar, 'valid_max', '') != '', 'valid_max should exist and not be empty or valid_range should be defined')
```

Comment:
Is this intentionally only run when ncvar does not have `standard_name`?

Reply:
This should actually be calling the function I defined instead.